code/modules/research/xenobiology/crossbreeding/stabilized.dm

```diff
@@ -45,57 +45,76 @@ Stabilized extracts:
 		S.linked_extract = src
 		STOP_PROCESSING(SSobj,src)
 
+
+
 //Colors and subtypes:
 /obj/item/slimecross/stabilized/grey
 	colour = "grey"
+	effect_desc = "Makes slimes friendly to the owner"
 
 /obj/item/slimecross/stabilized/orange
 	colour = "orange"
+	effect_desc = "Passively ties to increase or decrease the owner's body temperature to normal"
 
 /obj/item/slimecross/stabilized/purple
 	colour = "purple"
+	effect_desc = "Provides a regeneration effect"
 
 /obj/item/slimecross/stabilized/blue
 	colour = "blue"
+	effect_desc = "Makes the owner immune to slipping on water, soap or foam. Space lube and ice are still too slippery."
 
 /obj/item/slimecross/stabilized/metal
 	colour = "metal"
+	effect_desc = "Adds a sheet of material to a stack in the owners backpack, it is based on a timer."
```

Comment:
Every 30 seconds, adds a sheet of material to a random stack in the owner's backpack.